### ISSUE TITLE
Add Prettier for code formatting.

### ISSUE THREAD

**Issue by xwang345** [ywpark1/portfolio-generator#13]
Hi,

I just want to add Prettier to format the code.

**Comment by ywpark1** [comment]
Thanks,

Are you thinking to use Prettier for C# code?

**Comment by xwang345** [comment]
Yes. Prettier can reformat HTML CSS files but not for C#. we could include prettier in this project. See what the prettier supports:
![prettier](https://user-images.githubusercontent.com/23272523/48598503-2cdab480-e931-11e8-8b3a-d6038cac7627.png)

**Comment by ywpark1** [comment]
Thanks for providing information about that.
